mongodbatlas/resource_mongodbatlas_alert_configuration.go

```diff
@@ -100,6 +100,20 @@ func resourceMongoDBAtlasAlertConfiguration() *schema.Resource {
 						"units": {
 							Type:     schema.TypeString,
 							Optional: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								"DATADOG",
+								"EMAIL",
+								"FLOWDOCK",
+								"GROUP",
+								"OPS_GENIE",
+								"ORG",
+								"PAGER_DUTY",
+								"SLACK",
+								"SMS",
+								"TEAM",
+								"USER",
+								"VICTOR_OPS",
+								"WEBHOOK"}, false),
```

Comment:
This is the wrong array of strings, check the old code, should be:
"RAW",
								"BITS",
								"BYTES",
								"KILOBITS",
								"KILOBYTES",
								"MEGABITS",
								"MEGABYTES",
								"GIGABITS",
								"GIGABYTES",
								"TERABYTES",
								"PETABYTES",
								"MILLISECONDS",
								"SECONDS",
								"MINUTES",
								"HOURS",
								"DAYS"}, false),